src/vivarium_census_prl_synth_pop/results_processing/generate_names.py

```diff
@@ -0,0 +1,63 @@
+import glob
+import os
+from pathlib import Path
+from typing import Tuple
+
+import pandas as pd
+from loguru import logger
+
+from vivarium_census_prl_synth_pop.components.synthetic_pii import NameGenerator
+
+COLUMNS_FOR_NAME_PROCESSING = [
+    "first_name_id",
+    "middle_name_id",
+    "last_name_id",
+    "date_of_birth",
+    "sex",
+    "race_ethnicity",
+]
+
+
+def load_data(raw_results_dir: Path):
+    data = pd.DataFrame()
+
+    observer_directories = [f.path for f in os.scandir(raw_results_dir) if f.is_dir()]
+    for obs_dir in observer_directories:
+        logger.info(f'Processing data for {obs_dir.split("/")[-1]}...')
+        results = glob.glob(os.path.join(obs_dir, "*.csv.bz2"))
+        for file in results:
+            tmp = pd.read_csv(file)
+            data = pd.concat([data, tmp])
```

Comment:
Note: This is going to create a non-unique index. Haven't read far enough yet to know if it matters.